subgraphs/_reference_/src/sdk/protocols/generic/pool.ts

```diff
@@ -56,24 +56,31 @@ export class Pool {
     this.protocol = protocol;
     this.tokens = tokens;
     this.snapshoter = new PoolSnapshot(pool, protocol.event);
+    this.pool.lastUpdateTimestamp = protocol.event.block.timestamp;
```

Comment:
It gives a null error for some fields which are not set, but the value for lastUpdateTimestamp gets saved later because the value is getting updated. 

Fo your reference this is the error we get:
```A non-deterministic fatal error occured at block 6072093: failed to process trigger: block #6072093 (0x526a…683d), transaction 829353dcbe03e9fd7772930dc20768f9eb4631ebe84ff377604e68fd5887dfb6: Entity Pool[0x307833653262343530633063343939643833303131343633363736383065303637636430303964623933]: missing value for non-nullable field `isLiquidityToken` wasm backtrace: 0: 0x666e - <unknown>!generated/schema/Pool#save 1: 0x6ca5 - <unknown>!src/sdk/protocols/generic/pool/Pool#constructor 2: 0x8d77 - <unknown>!src/common/initializers/createOrUpdatePool 3: 0x966a - <unknown>!src/mappings/bondingManager/handleTranscoderActivated```